### DISCUSSION TITLE
[Kubernetes tutorial] map config to docker-compose

### DISCUSSION THREAD

**Discussion by berryp** [cuelang/cue#1001] [Q&A]
I've build a bunch of services based on the Kubernetes Tutorial, which is working great. However, soon I will have a need to support deploying via docker-compose for some environments. In the README for the tutorial is the line:

```
8. map a Kubernetes configuration to docker-compose (TODO)
```

Is this on your roadmap?

**Comment by myitcv** [comment]
Not as far as I am aware, at least not in the short term. Perhaps @mpvl can provide some context on the point itself, and then others might be able to contribute this section?

**Reply by berryp** [reply]
OK, thanks. I know it’s a small thing and I’m able to put something together myself, but it would be interesting to see what official docs come up with compared to my attempts. The manual Kubernetes cue files have been very educational and useful for me.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/1001.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
